content/docs/studio/index.md

```diff
@@ -1,25 +1,26 @@
 # DVC Studio
 
 [`DVC Studio`](https://studio.iterative.ai/) is a comprehensive and interactive
-collaboration tool for your Machine Learning projects.
+web application to manage and collaborate on your Machine Learning projects
+visually.
 
 Using the power of leading open-source tools DVC, CML and Git, it enables you to
 seamlessly manage data and models, run and track experiments, and visualize and
 share results.
 
 Project website: https://studio.iterative.ai
 
-![](https://static.iterative.ai/img/studio/main.png) _DVC Studio experiments
+![](https://static.iterative.ai/img/studio/main_v2.png) _DVC Studio experiments
 dashboard_
 
 Use DVC, CML and Studio to:
 
-- **Visualize, collaborate, and do everything that a regular ML tracking tool
-  does.**
+- Visualize, collaborate, and do everything that a regular ML tracking tool
+  does.
 
-- **Keep your code, data and model connected at all times.**
+- Keep your code, data and model connected at all times.
 
-- **Use the power of Git to track and preserve all your experiments.**
+- Use the power of Git to track and preserve all your experiments.
 
-- **Run [CI/CD](https://en.wikipedia.org/wiki/CI/CD) for your ML projects on
-  cloud resources of your choice without any new tools.**
+- Run [CI/CD](https://en.wikipedia.org/wiki/CI/CD) for your ML projects on cloud
```

Comment:
is this related to CML or is it a Studio feature?

Reply:
Studio, by virtue of CML. That is why the title for this section is "Use DVC, CML and Studio to:"